layers.go

```diff
@@ -423,6 +423,15 @@ func (r *layerStore) Load() error {
 		}
 	}
 
+	info, statErr := os.Stat(r.layerspath())
```

Comment:
… shouldn’t this capture the `info.ModTime` of the file _no later_ than at the time we are actually reading the data? I.e. either move this before the `ioutil.ReadFile(rpath)` at the top, or possibly micro-optimize that to an Open + Fstat + ReadAll + Close.

Reply:
Filed https://github.com/containers/storage/pull/1331 for this.